<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>403</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Block Wizard</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" colspan="4">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Insert a block into a book for a range of dates.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Start Date</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDateEdit" name="start_dateEdit">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>0</height>
      </size>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>202</width>
       <height>21</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>End Date (inclusive)</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QDateEdit" name="end_dateEdit">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>0</height>
      </size>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>202</width>
       <height>21</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Time</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>202</width>
       <height>21</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Length</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <spacer name="horizontalSpacer_5">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>202</width>
       <height>22</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Text to Apply</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="3">
    <widget class="QLineEdit" name="lineEdit"/>
   </item>
   <item row="7" column="0" colspan="4">
    <widget class="QGroupBox" name="day_groupBox">
     <property name="title">
      <string>Days to Apply</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0" colspan="4">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="1">
    <widget class="QFrame" name="time_frame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QGroupBox" name="clinicians_groupBox">
     <property name="title">
      <string>Clinicians</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QSpinBox" name="spinBox">
     <property name="suffix">
      <string> minutes</string>
     </property>
     <property name="maximum">
      <number>300</number>
     </property>
     <property name="singleStep">
      <number>5</number>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="9" column="0" colspan="4">
    <widget class="QLabel" name="progress_label">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="4">
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
